auto-discovery/kubernetes/README.md

```diff
@@ -135,11 +135,13 @@ kubectl -n juice-shop annotate service juice-shop auto-discovery.securecodebox.i
 | config.apiVersion | string | `"config.securecodebox.io/v1"` |  |
 | config.cluster.name | string | `"docker-desktop"` |  |
 | config.containerAutoDiscovery.enabled | bool | `false` |  |
-| config.containerAutoDiscovery.scanConfig.annotations | object | `{}` | annotations to be added to the scans started by the auto-discovery |
-| config.containerAutoDiscovery.scanConfig.labels | object | `{}` | labels to be added to the scans started by the auto-discovery |
+| config.containerAutoDiscovery.scanConfig.annotations | object | `{}` | annotations to be added to the scans started by the auto-discovery all annotation values support templating |
+| config.containerAutoDiscovery.scanConfig.labels | object | `{}` | labels to be added to the scans started by the auto-discovery all label values support templating |
```

Comment:
The multi-line comments from values.yaml are rendered strangely here (and in the other READMEs). May want to give that another pass.

Reply:
Hey @malexmave, yeah good point.
I've reformatted the descriptions so that the templating note is seperates by the original description with a comma.
I've left the descriptions which end in a url without the comma as a link followed by a space and a comma look really weird.

Reply:
LGTM!